app/javascript/advertisements/helpers.js

```diff
@@ -0,0 +1,23 @@
+export const elementVisible = element => {
+  if (!element) return false
+  while (element) {
+    const style = getComputedStyle(element)
+    if (style.visibility === 'hidden' || style.display === 'none') return false
+    element = element.parentElement
+  }
+  return true
+}
+
+export const closeCodefund = (codefundElementId = 'codefund') => {
+  sessionStorage.setItem('codefund', 'closed')
+  document.getElementById(codefundElementId).remove()
+}
+
+export const codefundClosed = () =>
```

Comment:
Pedantic and more of a question:

Wouldn’t this variable be better written as `getCodeFundClosed` and the above as `setCodeFundClosed`?

Reply:
Not really a setter in the pure sense, but I agree that current impl names aren’t great.